test/test_core_extractor.py

```diff
@@ -10,22 +10,24 @@
 
 
 class SimpleTest(unittest.TestCase):
-    """Class to run unit test cases on the function definition extractor"""
+    """Class to run unit test cases on the function definition extractor test"""
     src_files = os.path.join(TestResource.tst_resource_folder, "test_repo", "src")
     file_path = (os.path.join(os.path.dirname(__file__), os.pardir)).split("test")[0]
 
     def test_get_file_names(self):
         """Function to test get_file_names method"""
         files = get_file_names(self.src_files)
         expected = [os.path.join(self.src_files, "HelloController.java"), os.path.join(self.src_files, "test_c.c"),
-                    os.path.join(self.src_files, "test_repo.java"), os.path.join(self.src_files, "test_cpp_code.cpp")]
+                    os.path.join(self.src_files, "test_repo.java"), os.path.join(self.src_files, "test_cpp_code.cpp"),
+                    os.path.join(self.src_files, "Python_annot_file.py"),
+                    os.path.join(self.src_files, "Python_file.py")]
         self.assertEqual(expected.sort(), files.sort())
 
     def test_get_function_names(self):
         """Function to test get_function_names method"""
         func, line_num = get_function_names(os.path.join(self.src_files, "HelloController.java"))
-        expec_func = ['index1', 'index2', 'meth']
-        expec_line_num = ['61', '67', '29']
+        expec_func = ['meth', 'index1', 'index2']
```

Comment:
Suggestion: For doc string its is nice to have a brief description. Applicable across the test files